pkgs/applications/misc/inspectrum/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub, pkgconfig, cmake, fftwFloat, qt5 }:
+
+stdenv.mkDerivation rec {
+  name = "inspectrum-${version}";
+  version = "20160103";
+
+  src = fetchFromGitHub {
+    owner = "miek";
+    repo = "inspectrum";
+    rev = "a60d711b46130d37b7c05074285558cd67a28820";
```

Comment:
Is there a reason why you use the latest revision? We prefer to use stable releases in nixpkgs. As far as I can see the commits since the last release only affect OSX and you have specified linux as platform.